src/FunderMaps.Portal/Controllers/OopsController.cs

```diff
@@ -0,0 +1,46 @@
+using FunderMaps.Core.Types;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System.Net;
+
+namespace FunderMaps.Portal.Controllers
+{
+    // FUTURE: This is copied from FunderMaps.WebApi.Controllers.OopsController maybe move to FunderMaps.AspNetCore?
```

Comment:
Maybe centralize this controller? We use it in all three web projects

Reply:
Yes, however I do not know it that will work. I'll create a new PR